File: System/Common/BloodPressure.cs
using System;

namespace Common
{
    public class BloodPressure : Measurement
    {

        public long PatientID { get; set; }
        public int systolicPressure { get; set; }
        public int diastolicPressure { get; set; }


        public BloodPressure(long PatientID, string timestamp, int systolicPressure, int diastolicPressure)
        {
            this.PatientID = PatientID;
            this.Timestamp = timestamp;
            this.systolicPressure = systolicPressure;
            this.diastolicPressure = diastolicPressure;
        }


        public Boolean isCritical()
        {
            if (systolicPressure < 90 || systolicPressure > 140 || diastolicPressure < 60 || diastolicPressure > 90)
                return true;
            else
                return false;
        }
    }
}

File: System/Model/src/Model/BloodPressure.java
package Model;

public class BloodPressure extends Measurement {

    private int systolicPressure;
    private int diastolicPressure;

    public BloodPressure(long patientId, String timestamp, int systolicPressure, int diastolicPressure) {
        super(patientId, timestamp);
        this.systolicPressure = systolicPressure;
        this.diastolicPressure = diastolicPressure;
    }

    public int getSystolicPressure() {
        return systolicPressure;
    }

    public int getDiastolicPressure() {
        return diastolicPressure;
    }


    public boolean isCritical() {
        return systolicPressure < 90 || systolicPressure > 140 || diastolicPressure < 60 || diastolicPressure > 90;
    }

    public String toString() {
        return super.getTimestamp() + "	" + systolicPressure + "/"
                + diastolicPressure;
    }
}
